### PR TITLE
add revenuePrecision to MaxAd

### PR DESCRIPTION
**Pull request by uditrugman**
This patch adds revenuePrecision to MaxAd structure using the documented getRevenuePrecision API.

I'd appreciate it if you could add it to your main branch.

Thanks

### CODE REVIEW

File: applovin_max/lib/src/ad_classes.dart [hunk 1]
```diff
@@ -9,6 +9,9 @@ class MaxAd {
   /// The ad’s revenue amount, or 0 if no revenue amount exists.
   final double revenue;
 
+  /// The precision of the revenue as defined in the docs. possible values: 'publisher_defined', 'exact', 'estimated', 'undefined', ''
```
Comment (alhiwatan) [inline applovin_max/lib/src/ad_classes.dart:12]:
```suggestion
  /// The precision of the revenue value for this ad.
  ///
  /// Possible values:
  /// * "publisher_defined" - If the revenue is the price assigned to the line item by the publisher.
  /// * "exact" - If the revenue is the resulting price of a real-time auction.
  /// * "estimated" - If the revenue is the price obtained by auto-CPM.
  /// * "undefined" - If we do not have permission from the ad network to share impression-level data.
  /// * "" - An empty string, if revenue and precision are not valid (for example, in test mode).
  ///
```

**Comment by ycv005** [comment]
@thomasmso @alhiwatan Please take action on the PR.

**Comment by alhiwatan** [comment]
Has been merged on behalf of the original submitter.

**Comment by thomasmso** [comment]
Released in v3.3.0: https://github.com/AppLovin/AppLovin-MAX-Flutter/releases/tag/release_3_3_0

**Pull request closed without merging**